client/src/api/tournament/request.ts

```diff
@@ -25,4 +39,30 @@ export class TournamentAPI {
       throw e;
     }
   }
+
+  async addSignup(tournamentRoundDateId: number, inviteId: number): Promise<void> {
```

Comment:
could merge these into a single function that adds a parameter `action=add|remove` that then posts to  `/tournament/signup/${action}`

Reply:
Good call